README.md

```diff
@@ -1 +1,68 @@
-# React-Capstone-project-Financial-modeling
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto currencies
+
+In this project I have built a Crypto currency app, you can see the common currencies on the home page with some details about them, once you click on the arrow, it will take you to another page with all the information for the selected currency. All the data are fitched from binance Api.
+
+
+## Built With
+
+- Javascript
+- React
+- Redux
+
+## Live Demo
+[Live Demo](https://taupe-genie-b9016f.netlify.app/)
+
+## Loom Video
+[Video](https://www.loom.com/share/9daa1bf4c32a4b0592922403d65e6823)
+
+## Getting Started
+
+To successfully utilize this project first of all you need to clone or download the repository and follow the setup process.
+
+## Setup
+
+- copy this link
+
+https://github.com/tarek102/react-capstone-project-financial-modeling.git
+
+- Choose the directory that you want to clone the repository.
+- Open the terminal in this directory and type the following command
+
+``` git clone https://github.com/tarek102/react-capstone-project-financial-modeling.git ```
+
+- run  ``` npm install ```
+- run ``` npm start ```
+
+
+## prerequisites:
+- Visual Studio
+- Browser
+- Github
+- Node JS
+
+
+## Author
+
+👤 Tarek
+- GitHub: [@tarek102](https://github.com/tarek102)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/tarek102/react-capstone-project-financial-modeling/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to [etcNelson Sakwa](https://www.behance.net/sakwadesignstudio)  
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you fix this broken link error when clicking on the `[MIT](./LICENSE)` link. This is because there is no `LICENSE` file in the **root directory**. You can fix this by only adding a file name `LICENSE` with the **[following content](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** in your root directory.